cmd/installer/cli/signature_test.go

```diff
@@ -0,0 +1,126 @@
+package cli
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/*
+var testdata embed.FS
+
+func Test_verifySignature(t *testing.T) {
+	tests := []struct {
+		name       string
+		license    string
+		wantErr    bool
+		wantErrMsg string
+	}{
+		{
+			name: "valid signature",
+			license: func() string {
+				b, err := testdata.ReadFile("testdata/valid.yaml")
+				if err != nil {
+					t.Fatal(err)
+				}
+				return string(b)
+			}(),
+			wantErr:    false,
+			wantErrMsg: "",
+		},
+		{
+			name: "valid signature without entitlement signature",
+			license: func() string {
+				b, err := testdata.ReadFile("testdata/valid-without-entitlement-signature.yaml")
+				if err != nil {
+					t.Fatal(err)
+				}
+				return string(b)
+			}(),
+			wantErr:    false,
+			wantErrMsg: "",
+		},
+		{
+			name: "valid signature with entitlement signature",
+			license: func() string {
+				b, err := testdata.ReadFile("testdata/valid-with-entitlement-signature.yaml")
+				if err != nil {
+					t.Fatal(err)
+				}
+				return string(b)
+			}(),
+			wantErr:    false,
+			wantErrMsg: "",
+		},
+		{
+			name: "invalid signature",
+			license: func() string {
+				b, err := testdata.ReadFile("testdata/invalid-signature.yaml")
+				if err != nil {
+					t.Fatal(err)
+				}
+				return string(b)
+			}(),
+			wantErr:    true,
+			wantErrMsg: "failed to verify license signature: signature is invalid: crypto/rsa: verification error",
+		},
+		{
+			name: "licenseID field changed",
+			license: func() string {
+				b, err := testdata.ReadFile("testdata/invalid-changed-licenseID.yaml")
```

Comment:
i don't think adding a dedicated license for every test case is feasible here. why not just keep one of them to make sure we are validating changed fields, and then simply write a unit test for the `verifyLicenseData` function which doesn't require a valid signature?

Reply:
which code was copied from kots? i'm not suggesting changes to the code itself, just the tests.